databases/week1/week1.sql

```diff
@@ -0,0 +1,32 @@
+--1)Find out how many tasks are in the task table
+  SELECT count(id) AS count_task  FROM task;
+
+--2)Find out how many tasks in the task table do not have a valid due date
+SELECT count(id) AS  "count_tasks_with_invalid_due_date" FROM task WHERE due_date IS null;
```

Comment:
Just as a note: You don't need the quotes around the alias when you don't have any spaces in the name

```suggestion
SELECT count(id) AS  count_tasks_with_invalid_due_date FROM task WHERE due_date IS null;
```